lua/entities/gmod_wire_expression2/core/custom/cl_prop.lua

```diff
@@ -10,7 +10,9 @@ E2Helper.Descriptions["propSpawn(ean)"] = "Rotation, Frozen Spawns a prop with t
 E2Helper.Descriptions["propSpawn(svan)"] = "Model path, Position, Rotation, Frozen Spawns a prop with the model denoted by the string file path, at the position denoted by the vector, and rotated to the angle given. If frozen is 0, then it will spawn unfrozen."
 E2Helper.Descriptions["propSpawn(evan)"] = "Position, Rotation, Frozen Spawns a prop with the model of the template entity, at the position denoted by the vector, and rotated to the angle given. If frozen is 0, then it will spawn unfrozen."
 E2Helper.Descriptions["seatSpawn(sn)"] = "Model path, Frozen Spawns a prop with the model denoted by the string filepath. If frozen is 0, then it will spawn unfrozen."
+E2Helper.Descriptions["seatSpawn(sns)"] = "Last string refers to seat type, Valid types are Seat_Airboat, Seat_Pod and Seat_Jeep"
```

Comment:
Pretty sure there are more valid types than those: https://github.com/Facepunch/garrysmod/blob/master/garrysmod/lua/autorun/base_vehicles.lua

Should also explain that it's "Animation" or something

Reply:
i actually tried to search the github references for this in the actual wiki forthe "PlayerVehicleSpawn" of this and their were non, and oddly enough even this one gives different uses than what we do